### ISSUE TITLE
Headings with no content identified by screen readers

### ISSUE THREAD

**Issue by imperator99** [govau/dta-gov-au#285]
# What’s the issue?

On the bottom of landing pages as well as the accessibility statement, a screen
reader only heading ‘Related content’ is not visible but is still being read out to screen readers. There is no related content that follows this heading.

# Why it matters

This creates excessive noise for screen reader users, as well as provide incorrect
context of information that is to follow (footer navigation).

# Our recommendation

Remove the ‘Related content’ heading.

**Comment by imperator99** [comment]
Resolved by https://github.com/govau/dta-gov-au/pull/298 and https://github.com/govau/dta-website-rebuild/pull/375.
